apps/whale-api/src/module.indexer/model/dftx/composite.swap.spec.ts

```diff
@@ -32,6 +32,15 @@ beforeEach(async () => {
   }
 
   await container.generate(1)
+
+  app = await createTestingApp(container)
+  const defiCache = app.get(DeFiDCache)
+
+  const tokenResult = await container.call('listtokens')
+  // precache
```

Comment:
Curious if this is needed?

Reply:
nope.. i think i messed up something previously.. removed